test/zero-downtime-flagd-proxy/README.md

```diff
@@ -0,0 +1,28 @@
+# FlagD Proxy Zero downtime test
+
+## How to run
+
+Clone this repository and run the following command:
+
+```shell
+FLAGD_PROXY_IMG=your-flagd-image FLAGD_PROXY_IMG_ZD=your-flagd-second-image ZD_CLIENT_IMG=your-zd-client-image make run-zd-test
+```
+
+This will create a flagd-proxy and a job in `flagd-zd-test` namespace,
+where the test will be run.
+
+Please be aware, you need to build your custom image for the zd-client
+and two images for flagD first.
+
+To build your images using [ko](https://github.com/ko-build/ko),
+you need to login to your repository, where the images will be pushed:
```

Comment:
We can surely create a Dockerfile, it will be very simple, but since it's not a "production software image" I choose `ko` as it does not need  any other additional file to build and it's quite straight-forward to use. I might think that in the future we will adapt also the rest of the images, as it has some advantages to docker. You can find them [here](https://skaffold.dev/docs/builders/builder-types/ko/). 

But yes, for now we can still stick to docker and I can provide a Dockerfile :)